ts/features/itwallet/navigation/useItwLinkingOptions.tsx

```diff
@@ -22,7 +30,13 @@ export const useItwLinkingOptions = (): PathConfigMap<AppParamsList> => {
             "credential/issuance",
           [isItwValid
             ? ITW_ROUTES.DISCOVERY.ALREADY_ACTIVE_SCREEN
-            : ITW_ROUTES.DISCOVERY.INFO]: "discovery/info"
+            : ITW_ROUTES.DISCOVERY.INFO]: "discovery/info",
+          [presentationRoute]: {
+            path: "presentation/credential-detail/:credentialType?",
+            parse: {
+              credentialType: (credentialType: string) => credentialType
+            }
```

Comment:
Why not use a path param instead?

```suggestion
          [presentationRoute]: {
            path: "presentation/credential-detail/:credentialType",
```

This translates to `ioit://itw/presentation/credential-detail/MDL`. Shorter, cleaner. What do you think?